lib/public/AppFramework/Http/StreamGeneratorResponse.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * SPDX-FileCopyrightText: 2024 Nextcloud GmbH and Nextcloud contributors
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+namespace OCP\AppFramework\Http;
+
+use Generator;
+use OCP\AppFramework\Http;
+
+/**
+ * Class StreamResponse
+ *
+ * @since 31.0.0
+ *
+ * @template-extends Response<int, array<string, mixed>>
```

Comment:
```suggestion
 * @template-extends Response<Http::STATUS_*, array<string, mixed>>
```

To fix the psalm warning below.

Reply:
@kesselb you're amazing! You're title should be psalm master!